test/benchmarks/benchmark_SOR_update_method.jl

```diff
@@ -63,9 +63,9 @@ df_SOR_update = if !isfile(fn_SOR_update_benchmark_df) || recalculate
                 sim.detector, sim.electric_potential.grid, sim.medium, sim.electric_potential.data, sim.imp_scale.data, 
             ));
             ndrange = size(pcs.potential)[1:3] .- 2
-            dev = get_device(pcs.potential)
```

Comment:
Here, we should also use `_get_kabackend` rather than `get_device`
Add the beginning of the file, it also says: `using KernelAbstractions: get_device`, I guess this will not work anymore..

Not sure if this benchmarking scripts are used for the actual tests or to be run manually if wanted though.

Reply:
Ooops, overlooked that one, changed now.